frontend/src/App.js

```diff
@@ -1,15 +1,45 @@
 import React, {useState, useEffect} from 'react';
+import dotenv from "dotenv";
 import './App.css';
+import GitHubLogin from 'react-github-login';
+import axios from "axios";
```

Comment:
Please do not use axios on the client side, where we can use vanilla `fetch` instead, without any libraries. Thank you.